config/vufind/contentsecuritypolicy.ini

```diff
@@ -0,0 +1,30 @@
+; Settings for Content Security Policy header, you can learn more here:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+[CSP]
+; It is possible to use CSP only in report mode. See report_uri setting below.
```

Comment:
I wonder if it's going to be confusing to combine this setting with the fact that report_only is always true in development mode. Would it make sense to tie the configurations with the modes here more explicitly, so the user could set the behavior for development or production mode independently? For example, we could make this an array with `report_only[development]` and `report_only[production]` and then the code could test on something like `$this->config->CSP->report_only[APPLICATION_ENV] ?? false` to check the setting.

Reply:
I refactored it that way. It is much more transparent what is going on. Thanks for that suggestion.